### ISSUE TITLE
navbar-text is not correctly padded when used with .navbar-right inside navbar-fixed-top

### ISSUE THREAD

**Issue by jenscski** [twbs/bootstrap#16694]
This simple code shows the Text not correctly padded at the right.

```html
<nav class="navbar navbar-default navbar-fixed-top">
    <div class="container-fluid">
        <div class="navbar-header">
            <a class="navbar-brand" href="#/">Project name</a>
        </div>
        <div id="navbar" class="collapse navbar-collapse">
            <p class="navbar-text navbar-right">Text</p>
        </div>
    </div>
</nav>
```

**Comment by cvrebert** [comment]
@jenscski Please make the example into a JS Fiddle or JS Bin.

**Comment by jenscski** [comment]
Have created a JS Fiddle, http://jsfiddle.net/jenscski/jeem5wum/

Tested with navbar-fixed-bottom, and same error there.

**Comment by twbs-lmvtfy** [comment]
Hi @jenscski!

You appear to have posted a live example (http://fiddle.jshell.net/jenscski/jeem5wum/show/light/), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 50, column 9 thru line 50, column 58: Duplicate ID ```` navbar ````.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by jenscski** [comment]
JS Fiddle updated to remove validation errors http://jsfiddle.net/jenscski/jeem5wum/

**Comment by twbs-lmvtfy** [comment]
Hi @jenscski!

You appear to have posted a live example (http://fiddle.jshell.net/jenscski/jeem5wum/show/light/), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 50, column 9 thru line 50, column 58: Duplicate ID ```` navbar ````.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cvrebert** [comment]
After fixing the duplicate ID: http://jsfiddle.net/jeem5wum/2/

**Comment by cvrebert** [comment]
@mdo Is guess this is WontFix since it involves the navbar?

**Comment by mdo** [comment]
Yeah, not going to tweak that anymore in v3.
